great_expectations/cli/toolkit.py

```diff
@@ -68,7 +73,7 @@ def prompt_profile_to_create_a_suite(
 
 
 def get_or_create_expectation_suite(
-    expectation_suite_name: str,
+    expectation_suite_name: Union[str, None],
```

Comment:
@Kilo59 Would `expectation_suite_name: Optional[str]` not be an equivalent shorter version of the same?  Thanks.

Reply:
@Kilo59 I feel that diverse style is even more confusion.  I think that we either use `Optional` everywhere or use the `Union` with `None` everywhere.  To me, consistency is more important than which alternatives we adopt.  Hence, until we have agreed upon and implemented a consistent approach across the entire codebase, I would feel better about keeping `Optional`.  Thanks.